app/main.py

```diff
@@ -1,3 +1,68 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return "Distance: " + str(self.km) + " kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, new_distance):
+        if isinstance(new_distance, Distance):
+            return Distance(self.km + new_distance.km)
+        elif new_distance is not isinstance:
+            return self + Distance(new_distance)
+
+    def __iadd__(self, new_distance):
+        if isinstance(new_distance, Distance):
+            self.km += new_distance.km
+            return self
+        elif new_distance is not isinstance:
```

Comment:
what does this string mean?

Reply:
That's error. it's can be If Isinstance(new_distace, (int, float)) or else. sorry

Reply:
make it and push

Reply:
done